pom.xml

```diff
@@ -5,67 +5,37 @@
-    <module>components/forks/poi</module>
-    <module>components/forks/turbojpeg</module>
-    <module>components/autogen</module>
-    <module>components/formats-api</module>
-    <module>components/formats-bsd</module>
-    <module>components/formats-gpl</module>
-    <module>components/bundles/bioformats_package</module>
-    <module>components/bio-formats-plugins</module>
-    <module>components/bundles/loci-tools</module>
-    <module>components/metakit</module>
-    <module>components/ome-xml</module>
-    <module>components/bio-formats-tools</module>
-    <module>components/test-suite</module>
-    <module>components/stubs/lwf-stubs</module>
-    <module>components/stubs/mipav</module>
-    <module>components/xsd-fu</module>
-    <module>docs/sphinx</module>
-  </modules>
 
-  <properties>
-    <!-- If two artifacts on the classpath use two different versions of the
-         same dependency, behavior is inconsistent at best, and often broken.
-         The following properties facilitate consistency of dependency
-         versions between various projects in the SciJava software stack.
-         When possible, we advise using the relevant groupId and version
-         properties for your dependencies rather than hardcoding them. -->
-
-    <vcs.revision>${revision}</vcs.revision>
-    <release.version>5.3.0-SNAPSHOT</release.version>
-    <vcs.shortrevision>${shortrevision}</vcs.shortrevision>
-    <date>${maven.build.timestamp}</date>
-    <year>2016</year>
-    <project.rootdir>${basedir}</project.rootdir>
-    <imagej1.version>1.48s</imagej1.version>
-    <jgoodies-forms.version>1.7.2</jgoodies-forms.version>
-    <log4j.version>1.2.17</log4j.version>
-    <logback.version>1.1.1</logback.version>
-    <slf4j.version>1.7.6</slf4j.version>
-    <kryo.version>2.24.0</kryo.version>
-    <testng.version>6.8</testng.version>
-    <guava.version>17.0</guava.version>
-    <ome_common.version>5.3.0</ome_common.version>
-
-    <xsdfu.schemaver>2016-06</xsdfu.schemaver>
-    <xsdfu.schemapath>components/specification/released-schema/${xsdfu.schemaver}</xsdfu.schemapath>
-    <xsdfu.ome>${xsdfu.schemapath}/ome.xsd</xsdfu.ome>
+  <name>OME POI</name>
+  <description>Java API to handle Microsoft OLE 2 Compound Document format (Word, Excel). Based on poi-2.5.1-final-20040804.jar, with bugfixes for OLE v2 and memory efficiency improvements. Used by Bio-Formats for OLE support (cxd, ipw, oib, zvi). Used by VisBio overlays logic for XLS export feature.</description>
+  <url>http://jakarta.apache.org/poi/</url>
+  <inceptionYear>2008</inceptionYear>
+
+  <licenses>
+    <license>
+      <name>Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.openmicroscopy</groupId>
+      <artifactId>ome-common</artifactId>
+      <version>LATEST</version>
```

Comment:
Yeah I looked into alternatives and haven't found anything better as of yet, any other suggestions would certainly be welcome.